File: client/python/mesh/kinds/inbound.py
#
# Copyright (c) 2000, 2023, trustbe and/or its affiliates. All rights reserved.
# TRUSTBE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
#
#
from typing import Any, Sequence

from mesh.macro import index, serializable


@serializable
class Inbound:

    @index(0)
    def arguments(self) -> Sequence[Any]:
        return []

    @index(1)
    def attachments(self) -> {}:
        return {}

File: client/java/src/main/java/com/be/mesh/client/struct/Inbound.java
/*
 * Copyright (c) 2000, 2023, trustbe and/or its affiliates. All rights reserved.
 * TRUSTBE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
 *
 *
 */
package com.be.mesh.client.struct;

import com.be.mesh.client.annotate.Index;
import lombok.Data;

import java.io.Serializable;
import java.util.Map;

/**
 * @author coyzeng@gmail.com
 */
@Data
public class Inbound implements Serializable {

    private static final long serialVersionUID = -7034013150268079840L;
    @Index(0)
    private Object[] arguments;
    @Index(1)
    private Map<String, String> attachments;
}
